fli-iam/shanoir-ng
# [fli-iam#1643] Include the new OFSEP variables in the statistics export

- change stats export extension to .tsv (Tab Separated Value)
- add variables produced by the Kitware pipeline (see fli-iam#1643)
- add a last pipeline date, name, execution status

**How to test :**
- Export statistics
- Check that the file is now a .tsv (instead of a .txt)
- Check that a new column "execution" has been added
- Check that for datasets associated with an execution, the column "execution" is not empty, and that all properties are present in JSON
- Check that, for **new** execution, the device* fields are present in the JSON
    - derivedSequence : Type de séquence dérivée (null si séquence originale)
    - deviceConstructor : Fabricant de l’IRM
    - deviceMagneticField : Champ magnétique utilisé en Tesla
    - deviceModel : Model de l’IRM
    - deviceSerialNumber : Numéro de série de l’IRM
- Check that the SQL scripts are well ordered before merge
